### PR TITLE
Add history section and article for previous versions

### PR DESCRIPTION
**Pull request by cicdguy**
Adds a history section and an article to display historical documentation

### CODE REVIEW

File: vignettes/articles/website-versions.Rmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Previous Versions of Website"
```
Comment (bundfussr) [inline vignettes/articles/website-versions.Rmd:2]: Should we change it to "Previous and Development Versions of Website"?

File: _pkgdown.yml [hunk 2]
```diff
@@ -56,8 +60,8 @@ reference:
 
 navbar:
   structure:
-    left: [getstarted, reference, articles, news]
-    right: [search, slack, newissue, github]
+    left: [getstarted, reference, articles, news, versions]
```
Comment (bundfussr) [inline _pkgdown.yml:63]: Do we need to define the `versions` entry?

File: vignettes/articles/website-versions.Rmd [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+str_website_links <-
+  lapply(
+    X = all_version_folders,
+    FUN = function(x) {
+      x_label <- ifelse(x %in% "dev", "Development Site", x)
+      paste0("[", x_label, lst_tag_dates[[x]], "](", paste0(base_url, x), ")")
+    }
+  ) |>
+  unlist() |>
+  paste(collapse = "\n\n")
```
Comment (bundfussr) [inline vignettes/articles/website-versions.Rmd:79]: Should we have an admiraldev function for this code? Then maintaining it may be easier.

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: _pkgdown.yml (hunk 2 above)
Comment (bundfussr) [reply _pkgdown.yml:63]: When I build the website I get:
image
I would expect a "Versions" entry to the right of "Changelog".

File: vignettes/adrs_basic.Rmd [hunk 4]
```diff
@@ -872,4 +843,4 @@ dataset_vignette(
 
 ADaM | Sample Code
 ---- | --------------
-ADRS_BASIC | `r dyn_link("ad_adrs_basic.R", "https://github.com/pharmaverse/admiralonco/blob", "inst/templates/ad_adrs_basic.R", is_multiversion = TRUE)`
+ADRS_BASIC | [ad_adrs_basic.R](https://github.com/pharmaverse/admiralonco/blob/main/inst/templates/ad_adrs_basic.R)
```
[the remark this answers is not in the document]
Comment (bundfussr) [reply vignettes/adrs_basic.Rmd:846]: We should discuss it in the admiral core meeting today.

File: vignettes/articles/website-versions.Rmd (hunk 3 above)
Comment (ddsjoberg) [reply vignettes/articles/website-versions.Rmd:79]: Sounds like a great idea! 👯‍♂️

File: _pkgdown.yml (hunk 2 above)
Comment (ddsjoberg) [reply _pkgdown.yml:63]: I think this was a leftover from the example with the history link in two places and could be removed

File: vignettes/adrs_basic.Rmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (bundfussr) [reply vignettes/adrs_basic.Rmd:846]: It was decided to change to `` `use_ad_template("ADRS_BASIC")` ``.

**Review by bundfussr — APPROVED** [review]

**Pull request merged**